Makefile

```diff
@@ -21,18 +21,30 @@ BUILD_DATE  := $(shell date -u +"%Y-%m-%d-%H:%M")
 GIT_VERSION := $(shell git describe --match v --abbrev=8 --always --dirty)
 BRANCH_NAME := $(shell git rev-parse --abbrev-ref HEAD)
 VERSION     := $(GIT_TAG)
-# Go parameters
-GOCMD=go
-GOBUILD=$(GOCMD) build
+
+# Go parameters and builder
+GOVER ?= 1.9.1
+DOCKER_GO = docker run -it --rm -u $(USER) -w /go/src/$(GOMODULE) \
+    -v $(CURDIR)/.go:/go -v $(CURDIR):/go/src/$(GOMODULE) -v $${HOME}:/home/$(USER) \
+    -e GOOS=linux -e GOARCH=$(ARCH) -e CGO_ENABLED=0 -e BUILD=local $(GOBUILDER)
```

Comment:
I'm pretty sure you need CGO_ENABLED=1 here otherwise a few things in zedpac won't build

Reply:
Yes, it looks like we do; I missed that part.

It all is to call one function, which wraps some others. We should spend a few hours just replacing it with native go code and getting rid of cgo if we can.

Reply:
Done. Going to open a separate issue for `zedpac`.